remail/email_api/service.py

```
@@ -0,0 +1,530 @@
+        except (SMTPConnectError, exch_errors.ErrorConnectionFailed):
+            raise ee.SMTPServerConnectionFalse() from None
+        except SMTPDataError:
+            raise ee.SMTPDataFalse() from None
+        except SMTPSenderRefused:
+            raise ee.SMTPSenderFalse() from None
+        except SMTPServerDisconnected:
+            raise ee.SMTPServerConnectionFalse() from None
+        except (SMTPRecipientsRefused, exch_errors.ErrorInvalidRecipients):
+            raise ee.SMTPRecipientsFalse() from None
+        except Exception as e:
+            raise ee.UnknownError(f"An unexpected error occurred: {str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)